modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCachePreloaderAdapter.java

```diff
@@ -68,7 +68,7 @@ public GridCachePreloaderAdapter(CacheGroupContext grp) {
 
         log = ctx.logger(getClass());
 
-        finFut = new GridFinishedFuture();
+        finFut = new GridFinishedFuture<>(true);
```

Comment:
This future returns in **GridCachePreloaderAdapter#rebalanceFuture** and I think it should return **true** because rebalance is automatically completed successfully for it.